problemsets-mat223/linearalbegra.tex

```diff
@@ -2353,56 +2353,98 @@ \section*{Subspaces and Bases}
 			\begin{solution}
-				No. $\Span\{\vec u,\vec v\}$ is an infinite set of vectors
+				No. $\Span\Set{\vec u,\vec v}$ is an infinite set of vectors
 				which includes $\vec 0$, so it cannot be linearly independent and
 				therefore isn't a basis.
 			\end{solution}
 		\item What is the dimension of $V$?
 			\begin{solution}
-				A basis for $V$ has two vectors so it is two dimensional. We also
-				know this because $V$ is the $xy$-plane in $\R^3$ and all planes are two
-				dimensional.
+				A basis for $V$ has two vectors so it is two-dimensional. We also
+				know this because $V$ is the $xy$-plane in $\R^3$ and all planes 
+				are two-dimensional.
 			\end{solution}
 	\end{parts}
 
 
 	\question
 	Let $\vec a=\mat{1\\2\\3}$, $\vec b=\mat{4\\5\\6}$, $\vec c=\mat{7\\8\\8}$ (notice these vectors
 	are linearly independent) and 
-	let $P=\Span\{\vec a,\vec b\}$ and $Q=\Span\{\vec b,\vec c\}$.
+	let $P=\Span\Set{\vec a,\vec b}$ and $Q=\Span\Set{\vec b,\vec c}$.
 	\begin{parts}
 		\item Give a basis for and the dimension of $P$.
+			\begin{solution}
+				$\Set{\vec a,\vec b}$ is a basis for $P$, and so its dimension is 2. 
+			\end{solution}
 		\item Give a basis for and the dimension of $Q$.
+			\begin{solution}
+				$\Set{\vec b,\vec c}$ is a basis for $Q$, and so its dimension is 2. 
+			\end{solution}
 		\item Is $P\cap Q$ a subspace? If so, give a basis for it and its dimension.
+			\begin{solution}
+				Yes. $\Set{\vec b}$ is a basis for $P\cap Q$, and so its dimension is 1. 
+
+				To see that it's a subspace, note that if $\vec u,\vec v\in P\cap Q$
+				and $\alpha$ is a scalar, then $\vec u+\vec v,\alpha\vec u\in P$
+				since $P$ is a subspace, and $\vec u+\vec v,\alpha\vec u\in Q$ since $Q$ 
+				is a subspace subspaces. Therefore $\vec u+\vec v,\alpha\vec u\in P\cap Q$. 
+
+				Now if $\vec x\in P\cap Q$, we know that there are scalars 
+				$\alpha_1,\alpha_2,\beta_1,\beta_2$ such that
+				\[
+					\vec x=\alpha_1\vec a+\alpha_2\vec b 
+					\quad \text{and} \quad 
+					\vec x=\beta_1\vec b+\beta_2\vec c.
+				\]
+				Equating these and rearranging, we get
+				\[
+					\alpha_1\vec a=(\beta_1-\alpha_2)\vec b+\beta_2\vec c \in Q,
+				\]
+				from which it follows that $\alpha_1=0$, since $\vec a\notin Q$.
+				Therefore, $\vec x$ is a multiple of $\vec b$.
+				In other words, $P\cap Q=\Span\Set{\vec b}$. 
+			\end{solution}
 		\item Is $P\cup Q$ a subspace? If so, give a basis for it and its dimension.
+			\begin{solution}
+				No. For example $\vec a$ and $\vec c$ are both in $P\cup Q$, but 
+				$\vec a+\vec c \notin P\cup Q$. 
+
+				$\vec a+\vec c \notin P$ since if it was, we would also have
+				$(\vec a+\vec c)-\vec a=\vec c\in P$. We know this is impossible
+				since the vectors $\vec a,\vec b,\vec c$ are linearly independent,
+				and so $\vec c$ does not equal a linear combination of $\vec a$
+				and $\vec b$. 
+
+				An exactly analogous argument shows that $\vec a+\vec c \notin Q$. 
```

Comment:
-exactly

and spell out the definition of union. An alternative argument is:

I'd like something like: "A vector is in P\cup Q if it is in P or Q. v is in P if the equation $v=ta+sb$ is consistent and v is in Q if the equation $v=tb+sc$ is consistent. Since neither of these equations is consistent, v cannot be in P or Q, and is therefore not in P cup Q.

Reply:
I'm spelling out the definition of union.